knowledge-base/actions/setup-java/action-security.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Setup Java JDK'
+outbound-endpoints:
+  - fqdn: coveralls.io
+    port: 443
+    reason: "to send code coverage data"
+
+  - fqdn: github.com
+    port: 443
+    reason: "to query metadata"
+
+  - fqdn: jitpack.io
+    port: 443
+    reason: "to fetch artifacts from jitpack"
+
+  - fqdn: repo.maven.apache.org
+    port: 443
+    reason: "to fetch dependencies managed by Apache Maven"
```

Comment:
Please add the harden-runner-link at the end pointing to the link mentioned in the associated issue.